Integrate Kotlin bindings generation (#838)
matrix-org/matrix-rust-sdk: bindings/kotlin/README.md

# Matrix Rust SDK components for kotlin

> For now, his project only builds the matrix-sdk-crypto-ffi crate, generate the android bindings and wraps that as a aar lib distributed on maven.

Reviewer: ```suggestion
For now, this project only builds the matrix-sdk-crypto-ffi crate, generate the android bindings and wraps that as an aar lib distributed on maven.
```
